Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -270,11 +270,12 @@ function Get-CertificateFromDomainContollerToLocalFile {
         }
         $ParsedUrl = [System.Uri]$computerUrl
         if ($ParsedUrl.Port -lt 0 -OR $ParsedUrl.Host -eq "" -OR $ParsedUrl.Scheme -eq "") {
-            throw "Incorrect Url format entered from: $computerUrl" 
+            throw "Incorrect Url format entered from: $computerUrl. The correct Url format is protocol://host:port (Example: ldaps://yourserver.com:636)." 
         }
-        $ResultUrl = $ParsedUrl.Scheme + '://' + $ParsedUrl.Host + ':' + $ParsedUrl.Port
+        
+        $ResultUrl = $ParsedUrl.GetLeftPart('Authority')
```

Comment:
Does using string work? I'd expect the parameter to be enum, so `[UriPartial]::Authority`